docs/customization.rst

```diff
@@ -118,6 +123,34 @@ Unlike, newer versions of mkdocs-material theme, this theme also supports the us
 
         :hero: Configuration options to personalize your site.
 
+.. confval:: hide-navigation
+
+    This theme uses the ``:hide-navigation:`` metadata to hide the page's navigation menu (on
+    large enough viewports only).
+
+    .. code-block:: rst
+        :caption: Hide the navigation menu like so:
+
+        :hide-navigation:
+
+.. confval:: hide-toc
+
+    This theme uses ``:hide-toc:`` metadata to hide the page's Table of Contents (on large enough
```

Comment:
If specified, hides the local table of contents shown on the right side of the page.  By default the local table of contents is shown if the page contains sub-sections and the browser viewport is sufficiently wide.  If the toc.integrate feature is enabled, this option has no effect.